### DISCUSSION TITLE
Top10 intraday wert korrekt ?

### DISCUSSION THREAD

**Discussion by pietnie** [solectrus/solectrus#1888] [General]
Hallo, ich bin gerade darüber gestolpert, dass in meiner Top10-Darstellung zur Erzeugung vermutlich auch der Netzbezug mit angezeigt wird ? Kannst Du dies bitte einmal prüfen ?
![image](https://user-images.githubusercontent.com/111865951/229555466-5eed543f-1895-4824-a412-d9660befed62.png)
![image](https://user-images.githubusercontent.com/111865951/229555607-a9528f23-0535-4c3b-a5c8-6c4273232f26.png)

**Comment by ledermann** [comment]
Der Netzbezug wird nicht versehentlich eingerechnet, da bin ich mir recht sicher, auch wenn es hier von den Zahlen tatsächlich so aussieht (45,2 + 3,2 = 48,4). Ich vermute eher ein Caching-Problem oder eine nicht ganz so präzise InfluxDB-Abfrage, die zu einer ungewollten Tages-"Hochrechnung" führt. 

Könntest Du den Sachverhalt morgen erneut bei Dir prüfen? Ich nehme an, dann wird es korrekt sein.

**Reply by pietnie** [reply]
Danke für das Feedback, am nächsten Tag war alles korrekt mit leichten cachingproblemen, war wohl nur ein Zufall. Hauptsache historisch ist alles richtig

**Reply by ledermann** [reply]
Ja, für vergangene Zeiträume sehe ich auch keinen Fehler.
Dass die Anzeige **während** des Tages zu hoch ist, scheint ein Fehler zu sein, dafür habe ich mal ein Issue angelegt.
